subprojects/soak/src/integTest/groovy/org/gradle/launcher/daemon/DaemonPerformanceMonitoringCoverage.groovy

```diff
@@ -25,7 +25,8 @@ import static org.gradle.integtests.fixtures.daemon.JavaGarbageCollector.ORACLE_
 
 class DaemonPerformanceMonitoringCoverage {
     static final def ALL_VERSIONS = [
+        new FullyQualifiedGarbageCollector(vendor: JvmVendor.KnownJvmVendor.ADOPTOPENJDK, version: JavaVersion.VERSION_15, gc: ORACLE_G1),
```

Comment:
I've added the latest JDK here, since I don't have Oracle JDK installed and also because we ought to be testing something recent.

Reply:
We need some sort of mechanism to keep these up-to-date.